recipes/libhugetlbfs/build.sh

```diff
@@ -0,0 +1,18 @@
+if [[ ${ARCH} == "64" ]]; then
+    export ARCH=x86_64
+fi
+
+# Remove the static library from being installed
+sed -i 's/libhugetlbfs.a//g' Makefile
+
+# We only bulid native, so we want this to go to the correct location
+# The location is lib, nomatter what
+export LIB64=lib
+export LIB32=lib
```

Comment:
Should these be in `PREFIX` or are they already prefaced with this?

```suggestion
export LIB64="${PREFIX}/lib"
export LIB32="${PREFIX}/lib"
```

Reply:
From the makefile
```
REALLIB32 = $(realpath $(PREFIX)/$(LIB32))
REALLIB64 = $(realpath $(PREFIX)/$(LIB64))
```

```
LIBDIR32 = $(PREFIX)/$(LIB32)
LIBDIR64 = $(PREFIX)/$(LIB64)
```


```
obj32/install:
	@$(VECHO) INSTALL-LIB32 $(LIBDIR32)
	$(INSTALL) -d $(DESTDIR)$(LIBDIR32)
	$(INSTALL) $(INSTALL_OBJ_LIBS:%=obj32/%) $(DESTDIR)$(LIBDIR32)

obj64/install:
	@$(VECHO) INSTALL-LIB64 $(LIBDIR64)
	$(INSTALL) -d $(DESTDIR)$(LIBDIR64)
	$(INSTALL) $(INSTALL_OBJ_LIBS:%=obj64/%) $(DESTDIR)$(LIBDIR64)

```